apps/src/gamelab/GameLabSprite.js

```diff
@@ -20,6 +20,12 @@ module.exports.createSprite = function(x, y, width, height) {
     s.debug = true;
   }
 
+  // Now that sprite lab is running natively, we need a collision function that
+  // does not peek at the JS interpreter state.
+  // For now, we are only doing single sprite overlap detection, so this just wraps
+  // _collideWithOne and exposes it for spritelab.
+  s.nativeOverlap = nativeOverlap;
```

Comment:
Not sure I follow... how does this change with multiple sprite overlap?

Reply:
`nativeOverlap()` is a wrapper for `_collideWithOne()`. If we wanted to support group collisions, I think we would need to refactor `_collideWith()` to not peek at the JS interpreter state so that it would work natively. Moved the comment to below to make it more clear which piece would have to change to support group overlap.

As an aside, I don't think supporting group collisions will be necessary anyways, because we want to keep track of which sprites were involved in the collision. The best way I know of to do that is to loop through each pair of sprites and check for individual collision. I could be wrong though